src/styles/blocks/clock.scss

```
@@ -0,0 +1,45 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  transform: translate(40vw, 30vh);
+  @include circle ($clock-diametr);
+
+  border: 1px solid black;
+  border-radius: 50%;
+  box-shadow: 10px 5px 5px #aaa;
+
+  &__minute-hand {
+    position: absolute;
+    background-color: #00f;
+    height: $minutes-hand-length;
+    width: 2px;
+    box-shadow: 10px 5px 5px #aaa;
+    margin-bottom: $minutes-hand-length;
+    transform-origin: bottom;
+    animation: turn 60s linear infinite ;
+  }
+
+  &__second-hand {
+    position: absolute;
+    background-color: #f00;
+    height: $secunds-hand-length;
+    width: 2px;
+    box-shadow: 10px 5px 5px #aaa;
+    margin-top: $minutes-hand-length - $secunds-hand-length;
+    margin-bottom: $minutes-hand-length;
+    transform-origin: bottom;
+    animation: turn 1000ms linear infinite;
```

And this one should show seconds
```suggestion
    animation: turn 60s linear infinite;
```